### ISSUE TITLE
letsencrypt certificate with multiple domains.

### ISSUE THREAD

**Issue by JokerGermany** [spantaleev/nextcloud-docker-ansible-deploy#19]
1. multiple domains at once
At the moment i am using nextcloud snap
When i use `sudo nextcloud.enable-https lets-encrypt`
i can put in multiple domains.

Is this with this container possible, too?

2. multiple certificates for multiple domains.
When you look into the certificate, you can see the other domains.
There are cases when you don't want this.
Is would be great if you could acquire multiple certificates.

**Comment by JokerGermany** [comment]
It looks like it's something which could be supported because in the setup_ssl.yml is:
`name: Determine domain**s** to obtain certificates for (Nextcloud)`

**Comment by spantaleev** [comment]
We obtain certificates on by one, which is more flexible, especially when it comes the time to retire some of the domains (but continue others). The Nextcloud playbook uses a `domains_to_obtain_certificate_for` variable, which is initialized in `setup_ssl.yml`. The playbook loops over that list of domains and obtains a certificate for each one (only one domain in each certificate). Currently, the Nextcloud playbook defines just 1 domain (`nextcloud.DOMAIN`) in that hardcoded list.

The Matrix playbook is more flexible - it uses a `matrix_ssl_domains_to_obtain_certificates_for` variable, which **merges** `matrix_ssl_additional_domains_to_obtain_certificates_for` (additional domains specified by the user) with the domains that the playbook detects to be in need of certificates.

We can make the Nextcloud playbook follow a similar model too.

**Comment by feydreva** [comment]
it would be nice if let's encrypt could grab multiple domain.
for example, my nextcloud instance serves 2 domains.
We could be able to set theses domain in the var.yml

**Comment by spantaleev** [comment]
Fixed by 1fcd15c.

Try something like:

```yaml
nextcloud_ssl_additional_domains_to_obtain_certificates_for:
- one.DOMAIN
- two.DOMAIN
```
